src/service.js

```diff
@@ -109,7 +110,7 @@ async function startService ({ peerId, port, peerAnnounceAddr, awsClient, connec
       service.handle(protocol, async ({ connection: dial, stream }) => {
         try {
           const connection = new Connection(stream)
-          const canceled = new LRU({ max: 200 })
+          const canceled = cancelsPerPeer.get(dial.remotePeer.toString())
```

Comment:
Could be null? Why not create here if not exists?

Reply:
I would say it is not possible. But we can fallback to make sure, in case libp2p triggers some weirdness events